src/modules/45-dashboards/components/AidaChatRenderer/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useRef } from 'react'
+import { Container, FlexExpander, Layout, Text } from '@harness/uicore'
+import { FontVariation } from '@harness/design-system'
+import { Message } from '@dashboards/types/AidaTypes.types'
+
+import css from './AidaChatRenderer.module.scss'
+
+interface AidaChatRendererProps {
+  messages: Message[]
+}
+
+const AidaChatRenderer: React.FC<AidaChatRendererProps> = ({ messages }) => {
```

Comment:
Might as well name import this like the other stuff.
```suggestion
const AidaChatRenderer: FC<AidaChatRendererProps> = ({ messages }) => {
```